vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -19,7 +19,6 @@ IMG_SKU="$6"
 #   Running hostname -f will fail on current AzureLinux AKS image. We don't not have active plan to resolve
 #   this for stable version and there is no customer issues collected. Ignore this failure now.
 CLOUD_INIT_LOG_MSG_IGNORE_LIST=(
```

Comment:
can we get rid of this var altogether if it'll just be empty?

Reply:
Sure I've removed the var